lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,164 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/views/fields',
+        'teams/js/models/team',
+        'text!teams/templates/edit-team.underscore'],
+       function (Backbone, _, gettext, FieldViews, TeamModel, edit_team_template) {
+           return Backbone.View.extend({
+
+               maxTeamNameLength: 255,
+
+               initialize: function(options) {
+                   this.courseId = options.teamParams.courseId;
+                   this.teamsUrl = options.teamParams.teamsUrl;
+                   this.topicId = options.teamParams.topicId;
+                   this.topicName = options.teamParams.topicName;
+                   this.languages = options.teamParams.languages;
+                   this.countries = options.teamParams.countries;
+
+                   this.eventAggregator = options.eventAggregator;
+                   _.bindAll(this, "cancelTeam", "createTeam");
+                   this.eventAggregator.bind("cancelTeam", this.cancelTeam);
+                   this.eventAggregator.bind("createTeam", this.createTeam);
+
+                   this.teamNameField = new FieldViews.TextFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Name (Required) *"),
+                       valueAttribute: 'name',
+                       bindEvents: false,
+                       helpMessage: gettext("The name that will identify your team")
+                   });
+
+                   this.teamDescriptionField = new FieldViews.TextareaFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Description (Required) *"),
+                       valueAttribute: 'description',
+                       editable: 'always',
+                       showMessages: false,
+                       bindEvents: false,
+                       descriptionMessage: gettext("A short description of the team to help other students understand the goals or directives the team is pursuing")
+                   });
+
+                   this.optionalDescriptionField = new FieldViews.ReadonlyFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Optional Characteristics"),
+                       valueAttribute: 'optional_description',
+                       helpMessage: gettext("You can help students find your team by specifying your team's characteristics. The more limitations you add, the fewer students may be interested in joining your group, so choose carefully!")
```

Comment:
"Help other learners decide whether to join your team by specifying some characteristics for your team. Choose carefully, because fewer people might be interested in joining your team if it seems too restrictive."
@explorerleslie Do we want to say anything about ability/inability to change name, description, characteristics once a team has been created?